internal/query/key.go

```diff
@@ -399,7 +399,7 @@ func (wm *PublicKeyReadModel) Query() *eventstore.SearchQueryBuilder {
 		Builder()
 }
 
-func (q *Queries) GetActivePublicKeyByID(ctx context.Context, keyID string, current time.Time) (_ PublicKey, err error) {
+func (q *Queries) GetActivePublicKeyByID(ctx context.Context, keyID string) (_ PublicKey, err error) {
```

Comment:
Its not really "GetActivePublicKeyByID" anymore more like "GetPublicKeyByID" right?
Otherwise we would need to add a grace period instead of a current time?

Reply:
Yes, you are right. Renamed.